src/csharp/Microsoft.Spark.Worker/Command/SqlCommandExecutor.cs

```diff
@@ -737,6 +735,23 @@ internal ArrowOrDataFrameGroupedMapCommandExecutor(Version version)
             return ExecuteArrowGroupedMapCommand(inputStream, outputStream, commands);
         }
 
+        private RecordBatch WrapArrowRecordBatchColumnsInAStruct(RecordBatch batch)
```

Comment:
I think we should either bring the if statement out to the caller or rename this method ?

Reply:
Why? I'm not opposed to it. What would you rename the method to? I don't have any strong opinions here

Reply:
To me the method name seems to imply that the RecordBatch will be wrapped in a Struct.  But that will happen if the version is >= 3.0, otherwise it is not wrapped in a struct.

Reply:
We can keep the name as is and @imback82 can provide feedback if needed.

Reply:
How about `WrapColumnsAsStructIfApplicable`?